src/server/main_service.h

```diff
@@ -52,6 +64,24 @@ class Service : public facade::ServiceInterface {
 
   facade::ConnectionStats* GetThreadLocalConnectionStats() final;
 
+  const CommandId* FindCmd(std::string_view cmd) const {
+    return registry_.Find(cmd);
+  }
```

Comment:
Why have this method?

Reply:
I'll remove it 🤷🏻 I only moved it because it polluted the header 😄

Reply:
Wow, I just found out it has a implementation with fixes in the .cc file, the header one was just never removed 😮

Reply:
To keep registry_ private & inaccessible

Reply:
:facepalm: 
right